src/dsp_tools/commands/excel2json/new_lists.py

```diff
@@ -75,7 +75,10 @@ def _parse_files(excelfolder: Path | str) -> list[ExcelFile]:
     file_names = [file for file in Path(excelfolder).glob("*list*.xlsx", case_sensitive=False) if _non_hidden(file)]
     all_files = []
     for file in file_names:
-        sheets = [ExcelSheet(sheet_name=name, df=df) for name, df in read_and_clean_all_sheets(file).items()]
+        sheets = [
+            ExcelSheet(excel_name=file.stem, sheet_name=name, df=df)
```

Comment:
why only the stem and not the entire `file.name`?

Reply:
Good point